tests/Aspire.Playground.Tests/README.md

```diff
@@ -0,0 +1,5 @@
+# Aspire.Playground.Tests
+
+Runs apps in `playground/` and hits some endpoints on them.
+
+Use `TEST_PLAYGROUND_APPHOST_FILTER` to run tests for specific playground apps. For example, `TEST_PLAYGROUND_APPHOST_FILTER=proxy`.
```

Comment:
what is `proxy`? Can we use a real app that works? Like TestShop?

Reply:
`ProxylessEndToEnd`. Yep, we can use `TestShop`.